testgrid/specs/full.yaml

```diff
@@ -1681,6 +1681,62 @@
     operatorVersion=$(kubectl get deployment -n rook-ceph rook-ceph-operator -o jsonpath='{.spec.template.spec.containers[0].image}')
     echo $operatorVersion | grep 1.5.12
 
+    k8sVersion=$(kubectl get nodes -o jsonpath='{.items[0].status.nodeInfo.kubeletVersion}')
+    echo $k8sVersion | grep 1.20
+- name: rook-upgrade-to-1.4-airgap
+  flags: "yes"
+  airgap: true
+  installerSpec:
+    kubernetes:
+      version: 1.19.x
+    containerd:
+      version: "latest"
+    weave:
+      version: "latest"
+    rook:
+      version: "latest"
```

Comment:
I think we should be explicit with the rook version. In this case `latest` = `1.0.4`

Reply:
good point, will change